src/main/scala/viper/gobra/reporting/VerifierError.scala

```diff
@@ -329,6 +329,13 @@ case class SpecImplementationPostconditionError(info: Source.Verifier.Info, spec
   override def localMessage: String = s"Postcondition of spec $specName might not hold"
 }
 
+case class PrivateEntailmentError(info: Source.Verifier.Info, specName: String, msg: String) extends VerificationError {
+  override def localId: String = "private_entailment_error"
+  override def localMessage: String = 
+    s"$msg of call ${info.trySrc[frontend.PInvoke](" ")}might not hold. " + 
+    s"Private specification might not entail the public specification."
```

Comment:
I reworked most of the error messages a few days ago (still needs to be updated), because they did not work as I wanted them to. I wanted it to call an error if the proof (the generated public method) failed, but I could not get the error message out of there. I think I created and implemented it inside the "private" library as:

`override def privateProofFunction(x: in.Function)(ctx: Context): Unit`

I at first thought that I can put the error message inside the generated proof, but It did not work, so now I changed it to:

`override def privateProofFunction(x: in.Function)(ctx: Context): Writer[Option[vpr.Method]]` 

Now I can get the error from the DefaultMethodEncoding by calling this "private" library.